hedy.py

```diff
@@ -1622,6 +1627,17 @@ def transpile_inner(input_string, level):
         else:
             invalid_command = invalid_info.command
             closest = closest_command(invalid_command, commands_per_level[level])
+            fixed_code = None
+            result = None
```

Comment:
@fRedelaar I fixed all tests. I think you will need to handle the case where fixed_code or result is none in the frontend/translations.

Reply:
Cool, great work! I've just sent a commit that made the bulb disappear whenever the code is fixed by the user. Or did you mean something else?

Reply:
I think Bjorn means the case happening when no fixed_code is given (but that should be ok for now)